nuxeo-coldstorage-web/package.json

```diff
@@ -0,0 +1,73 @@
+{
+  "name": "nuxeo-coldstorage-web",
+  "version": "0.1.0",
+  "description": "Nuxeo Cold Storage WebUI integration",
+  "author": "Nuxeo",
+  "license": "Apache-2.0",
+  "devDependencies": {
+    "@nuxeo/testing-helpers": "~3.0.2-rc.0",
+    "@open-wc/karma-esm": "^3.0.9",
+    "@rollup/plugin-node-resolve": "^7.1.3",
+    "chai": "^4.2.0",
+    "eslint": "^7.12.1",
+    "eslint-config-airbnb-base": "^14.2.0",
+    "eslint-config-prettier": "^6.15.0",
+    "eslint-plugin-html": "^6.1.0",
+    "eslint-plugin-import": "^2.22.1",
+    "husky": "^4.3.0",
+    "karma": "^5.2.3",
+    "karma-chrome-launcher": "^3.1.0",
+    "karma-coverage-istanbul-reporter": "^3.0.3",
+    "karma-firefox-launcher": "^2.1.0",
+    "karma-mocha": "^2.0.1",
+    "karma-mocha-reporter": "^2.2.5",
+    "karma-sauce-launcher": "^4.3.3",
+    "karma-sinon-chai": "^2.0.2",
+    "karma-source-map-support": "^1.4.0",
+    "karma-static": "^1.0.1",
+    "lint-staged": "^10.5.1",
+    "lit-html": "^1.3.0",
+    "mocha": "^8.2.1",
+    "polymer-cli": "^1.9.11",
+    "prettier": "2.1.2",
+    "rollup": "^2.33.1",
+    "rollup-plugin-copy": "^3.3.0",
+    "rollup-plugin-minify-html-literals": "^1.2.5",
+    "rollup-plugin-terser": "^7.0.2",
+    "sinon": "^9.2.1",
+    "sinon-chai": "^3.5.0"
+  },
+  "dependencies": {
+    "@nuxeo/nuxeo-elements": "~3.0.2-rc.0",
```

Comment:
Do you need anything specific from `3.0.2-rc.0` or any release candidate?

Also, please note that if master is supposed to target cloud, we should use `^` instead of `~`, so that it keeps getting cloud updates (where the minor is bumped). For LTS, you can keep `~` to keep benefiting from patches.